### PR TITLE
New documentation on IPythonMagics and getting started

### PR DESCRIPTION
**Pull request by a-larkins99**
## Content:
Adds both:
- GettingStarted.md
- IPythonMagics_commands.md

These add new guidance for setting up and starting with pace-python using Horace 3.6.3, as well as documentation on the new IPythonMagics commands following their migration to libpymcr. 

**Note:**  GettingStarted.md refers to pace-python `version: 0.3.0`. I have made the assumption that this is what the release will be called. Please update this as needed.

### CODE REVIEW

File: docs/GettingStarted.md
Comment (oerc0122) [inline docs/GettingStarted.md:5]:
```suggestion
Pace-Neutrons (formerly known as pace-python) is a Python package which brings Horace >3.6.3 and SpinW to Python, removing the need for Matlab whilst keeping the commands you have become familiar with.
```

File: docs/GettingStarted.md
Comment (oerc0122) [inline docs/GettingStarted.md:16]: Link to PyPI package

File: docs/GettingStarted.md
Comment (oerc0122) [inline docs/GettingStarted.md:31]:
```suggestion
with `m.`, transforming MATLAB arrays and cell-arrays to numpy `array`s/`list`s and `struct`s to `dict`s and dropping the semi-colon at the end of the line. For example,  
```

File: docs/GettingStarted.md
Comment (a-larkins99) [reply docs/GettingStarted.md:16]: We haven't done the full release yet. Duc wanted to wait until Horace 4 is released I believe.

File: docs/GettingStarted.md
Comment (oerc0122) [inline docs/GettingStarted.md:227]: Probably wants to be triple-quoted block

File: docs/GettingStarted.md
Comment (oerc0122) [inline docs/GettingStarted.md:199]: Link to other doc on magic

File: docs/GettingStarted.md
Comment (oerc0122) [reply docs/GettingStarted.md:16]: Doesn't matter too much, these docs will outlast this release.

**Review by mducle — APPROVED** [review]
Thanks. Looks good to me, ok to merge after addressing Jacob's comments.

File: docs/GettingStarted.md
Comment (a-larkins99) [reply docs/GettingStarted.md:16]: I can add the main pypi page then.

File: docs/GettingStarted.md
Comment (a-larkins99) [reply docs/GettingStarted.md:199]: How do you link to another doc? I don't know what the url will be to the thing once it is merged

Comment (a-larkins99) [reply docs/GettingStarted.md:199]: @oerc0122

**Review by oerc0122 — APPROVED** [review]

**Review by mducle — APPROVED** [review]

**Pull request merged**
